### PR TITLE
Parse new fields

### PR DESCRIPTION
**Pull request by kotmfu**
New fields to parse:

- `subNumber`
- `lotNumber`
- `authority`
- `depositTaken`

### CODE REVIEW

File: src/OpenRealEstate.Transmorgrifiers.RealEstateComAu/ReaXmlTransmorgrifier.cs [hunk 1]
```diff
@@ -670,8 +673,8 @@ private class SplitElementResult
             }
 
             // Land and CommericalLand should only provide lot numbers. 
-            var lotNumber = addressElement.ValueOrDefault("lotNumber");
-            var subNumber = addressElement.ValueOrDefault("subNumber");
+            listing.Address.LotNumber = addressElement.ValueOrDefault("lotNumber");
+            listing.Address.SubNumber = addressElement.ValueOrDefault("subNumber");
```
Comment (PureKrome) [inline src/OpenRealEstate.Transmorgrifiers.RealEstateComAu/ReaXmlTransmorgrifier.cs]: Can you fix the comment, above this please?

File: src/OpenRealEstate.Transmorgrifiers.RealEstateComAu/ReaXmlTransmorgrifier.cs [hunk 2]
```diff
@@ -745,11 +748,14 @@ private class SplitElementResult
...
             CalculateDisplayAddress(addressElement, listing.Address);
-            
+
             // Technically, the <municipality/> element is not a child of the <address/> element.
             // But I feel that it's sensible to still parse for it, in here.
             document.ValueOrDefaultIfExists(municipality => listing.Address.Municipality = municipality, "municipality");
 
+            // Street View
+            var streetView = addressElement.ValueOrDefault("streetView");
+
```
Comment (PureKrome) [inline src/OpenRealEstate.Transmorgrifiers.RealEstateComAu/ReaXmlTransmorgrifier.cs]: why this?

File: src/OpenRealEstate.Transmorgrifiers.RealEstateComAu/ReaXmlTransmorgrifier.cs [hunk 3]
```diff
@@ -1969,6 +1992,11 @@ private static int ConvertImageOrderToNumber(string imageOrder)
                                                     ? rentalPricing.RentalPrice.ToString("C0")
                                                     : null;
 
+                var depositTakenValue = rentElement.ValueOrDefault("depositTaken", "value");
+                rentalPricing.HasTakenDeposit = (string.IsNullOrEmpty(depositTakenValue)
+                                                    && depositTakenValue.ParseOneYesZeroNoToBool())
+                                                    || false;
```
Comment (PureKrome) [inline src/OpenRealEstate.Transmorgrifiers.RealEstateComAu/ReaXmlTransmorgrifier.cs]:
```suggestion
                rentalPricing.HasTakenDeposit = (string.IsNullOrEmpty(depositTakenValue) && 
                                                 depositTakenValue.ParseOneYesZeroNoToBool()) ||
                                                 false;
```

**Review by PureKrome — CHANGES_REQUESTED** [review]

File: src/OpenRealEstate.Transmorgrifiers.RealEstateComAu/ReaXmlTransmorgrifier.cs (hunk 2 above)
Comment (kotmfu) [reply src/OpenRealEstate.Transmorgrifiers.RealEstateComAu/ReaXmlTransmorgrifier.cs]: that was on the parse list but was removed, will remove

File: src/OpenRealEstate.Transmorgrifiers.RealEstateComAu/ReaXmlTransmorgrifier.cs (hunk 3 above)
Comment (kotmfu) [reply src/OpenRealEstate.Transmorgrifiers.RealEstateComAu/ReaXmlTransmorgrifier.cs]: no longer applies

**Review by PureKrome — APPROVED** [review]

**Pull request merged**